### PR TITLE
Alert avatar

### PR DESCRIPTION
**Pull request by deekshithmd**
completed alert and avatar components

### CODE REVIEW

File: components/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <img class="img-responsive img-round" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-xs">
+        <img class="img-responsive img-round" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-text-lg avatar-text img-round" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-md avatar-text img-round" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-sm avatar-text img-round" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-xs avatar-text img-round" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-lg">
```
Comment (anshulraheja) [inline components/avatar/avatar.html:27]: check if img-round class is adding come css in this avatar, as per there is no need to use this class here

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <div class="avatar avatar-sm">
+        <img class="img-responsive img-square" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-xs">
+        <img class="img-responsive img-square" src="../assets/images/avatar.png" alt="Avatar">
+    </div>
+    <div class="avatar avatar-text-lg avatar-text img-square" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-md avatar-text img-square" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-sm avatar-text img-square" role="img" alt="Avatar">AB</div>
+    <div class="avatar avatar-text-xs avatar-text img-square" role="img" alt="Avatar">AB</div>
```
Comment (anshulraheja) [inline components/avatar/avatar.html:42]: same, check for img-square also

**Review by anshulraheja — APPROVED** [review]
very good UI and code is nicely written.

File: components/avatar/avatar.html (hunk 1 above)
Comment (deekshithmd) [reply components/avatar/avatar.html:27]: yes, it is used to shape it round and to give border

File: components/avatar/avatar.html (hunk 2 above)
Comment (deekshithmd) [reply components/avatar/avatar.html:42]: same, to make it square and to give border

File: index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                </div>
+            </div>
+        </section>
+        <hr>
+        <!-- Installation Section -->
+        <section class="install-section" id="installation">
+            <div class="text-container">
+                <h1 class="margin-b">Installation</h1>
+                <p class="text-3xl">This UI can be used either by importing cdn in CSS file or by linking in the html file using link tag.</p>
+                <a href="documentation.html" class="btn btn-solid-primary link-btn link-style-none">Get Started</a>
```
Comment (dhimanaishwarya) [inline index.html]: you can add the correct link here

Comment (deekshithmd) [reply index.html]: ok, sure thank you for review

**Comment by deekshithmd** [comment]
> very good UI and code is nicely written.

Thank you...

**Comment by m26unkwn** [comment]
Stunning UI and COLORS.

**Comment by deekshithmd** [comment]
> Stunning UI and COLORS.

Thank you...

**Pull request merged**